sprint9/tema3.js

```diff
@@ -0,0 +1,23 @@
+// aici este o lista de obicte carti
+const carti = [
+    { titlu: "Hobbitul", autor: "J.R.R. Tolkien", esteCitită: true },
```

Comment:
```suggestion
    { titlu: "Hobbitul", autor: "J.R.R. Tolkien", esteCitita: true },
```

(optional): numele cheilor din obiect se doreste sa fie fara diacritice